packages/itwinui-css/backstop/tests/progress-indicator.html

```diff
@@ -225,148 +225,51 @@ <h2>Radial</h2>
         <h3>Determinate</h3>
 
         <p class="iui-text-small iui-text-muted">
-          For determinate radial progress indicators, percentage is set by
-          stroke-dashoffset: (88 - 88 * target / 100).
-          <br />
-          So 25% is stroke-dashoffset: 66; or stroke-dashoffset: calc(88 - 88
-          * 25 / 100);
+          For determinate variants, percentage is set by
+          <code>--iui-progress-percentage</code>.
         </p>
         <br />
 
         <div id="radial-size-large">
           <!-- Small determinate -->
```

Comment:
Doesn't necessarily relate to your changes, but this might be a good time to change the comments from "Small" to "Large" for all of the large determinate examples?

Reply:
done